src/openrct2/rct1.h

```diff
@@ -30,6 +30,12 @@
 #define RCT1_MAX_MAP_ELEMENTS       0xC000
 #define RCT1_MAX_SPRITES            5000
 #define RCT1_MAX_VEHICLES_PER_RIDE  12
+#define RCT1_MAX_STATIONS           4
```

Comment:
I have fixed this for the most part. There are a few I didn't switch over but that is because I am unsure of where to find the rct2 equivalent...If you see any left over that could have a RCT12 merge, let me know.